extensions/extensions-docker/aissemble-pipeline-invocation/pom.xml

```diff
@@ -24,6 +24,35 @@
             <version>${project.version}</version>
         </dependency>
     </dependencies>
+    <profiles>
+        <profile>
+            <id>ci</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>default-push</id>
+                                <configuration>
```

Comment:
S: Instead of duplicating the build_args, we should just move the config block below from the `default-build` execution to the plugin root.

Reply:
Unfortunately, the execution doesn't recognize the configuration after removing the execution id so will continue with this approach